docs/overview.rst

```diff
@@ -238,55 +237,53 @@ You have two choices:
 Current status and roadmap
 --------------------------
 
-At the moment of writing, HPy is still in its early stages of development. The
-following milestones have been reached:
+HPy is still in the early stages of development, but many big pieces are
+already in place. As on April 2021, the following milestones have been reached:
+
+  - one can write extensions which expose module-level functions, with all
+    the various kinds of calling conventions.
+
+  - there is support for argument parsing (i.e. the equivalents of
+    `PyArg_ParseTuple` and `PyArg_ParseTupleAndKeywords`).
+
```

Comment:
in the part about future work we should mention the "argument-clinic-like" way of specifying calling convention, which among the other things will allow the JITs and Cython to directly call the impl function, bypassing the argument parsing entirely